torch/nn/utils/rnn.py

```diff
@@ -26,7 +26,29 @@ class PackedSequence(PackedSequence_):
         batch_sizes (list[int]): list of integers holding information about
             the batch size at each sequence step
     """
-    pass
+
+    @classmethod
+    def _impute_data_prop(cls, name):
+        @property
+        def prop(self):
+            return getattr(self.data, name)
+        setattr(cls, name, prop)
+
+    @classmethod
+    def _impute_data_mask(cls, name):
+        """Impute method `name` of attribute `data` into class `cls`"""
+        def fn(self, *args, **kwargs):
+            return type(self)(getattr(self.data, name)(*args, **kwargs), self.batch_sizes)
+        setattr(cls, name, fn)
+
+# Impute following `torch.Tensor` methods and properties into `PackedSequence`
+_methods_to_impute = 'cuda cpu double float long int short char byte'.split()
```

Comment:
It would be simpler and more explicit to just write them out as a list

Reply:
Sure, we could still impute all the different cos/acos/sin/sin_ etc. methods, or should we write a wrapper for each one?